framework/integration/test/basic/basic_test.go

```diff
@@ -49,9 +49,27 @@ func Test_Finalizer_Integration_Basic(t *testing.T) {
 		t.Fatal("expected", nil, "got", err)
 	}
 
-	// We directly pass the configmap to UpdateFunc.
+	// We update the configmap with a meanless label to ensure a change in the
+	// ResourceVersion of the ConfigMap.
+	cm.SetLabels(
+		map[string]string{
+			"testlabel": "testlabel",
+		},
+	)
+	_, err = client.UpdateConfigMap(cm)
+	if err != nil {
+		t.Fatal("expected", nil, "got", err)
+	}
+
+	// We directly pass the _old_ configmap to UpdateFunc.
+	// This is expected to fail, we want to make sure that we only use the latest
+	// ResourceVersion of an object.
+	operatorkitFramework.UpdateFunc(createdConfigMap, createdConfigMap)
```

Comment:
This should atleast check in the integration test, that we no longer add multiple finalizers.

Reply:
We pass the old and the new value here. Only one is respected by the framework, which is the new one. Here we can pass nil for the old object because it is not of any interest. Would also be clearer for the people reading this code. Right now it looks more confusing.